synthea-hiv/uploader/uploader.py

```diff
@@ -64,8 +65,9 @@ def upload_openmrs_bundle(self, each_bundle: bundle.Bundle):
                         each_bundle.openmrs_patient.base.new_id)
 
       for openmrs_encounter in each_bundle.openmrs_encounters:
+        location = random.choice(list(locations.items()))
```

Comment:
I think a more realistic choice is to assign a location to each patient, then make the decision here based on the `subject` element of each `Encounter`. @kimaina what do you think?

Reply:
Either approach is ok for now. We do want that element of random allocation of location to encounters.  Approximately 30% of patients do switch clinics from time to time and the patient table is not always updated, instead encountr-location is always used for reporting purposes.

Reply:
Okay then let's leave it the way it is right now. @omarismail94 all my other comments minor, please feel free to merge after addressing them.